cluster/charts/rook-ceph/values.yaml

```diff
@@ -81,6 +81,7 @@ csi:
     #image: quay.io/k8scsi/csi-provisioner:v1.3.0
   #snapshotter:
     #image: quay.io/k8scsi/csi-snapshotter:v1.2.0
+  # attacher is required if Kubernetes version is less than 1.14
```

Comment:
instead of saying less than 1.14, we can say only for 1.13, right? CSI isn't supported pre-1.13.

Reply:
I will reframe this sentence